src/main.c

```diff
@@ -92,6 +92,39 @@ int main(int argc, char **argv)
 
 	lib_init(arguments.debug);
 
+	/* user tweaks or overrides */
+	FILE *f = fopen("/etc/clr-power-tweaks.conf", "r");
+	if (f) {
+		for (;;) {
+			char match[PATH_MAX];
+			char value[PATH_MAX];
+			char line[PATH_MAX];
+			int ret;
+
+			if (fgets(line, PATH_MAX, f) == NULL)
+				break;
+
+			ret = sscanf(line, "%s %s", match, value);
+			if (ret >= 1) {
+				/* comment */
+				if ((strlen(match) > 0) && (match[0] == '#')) {
+					continue;
+				};
+			};
+			if (ret == 1) {
+				if (arguments.debug)
+					fprintf(stderr, "Adding \"%s\" to exclusion list\n", match);
+				string_exclude(match);
+			};
+			if (ret == 2) {
+				write_string_to_files(match, value);
```

Comment:
Do we need the feature to add more things? That's what tmpfiles.d is for, which is what this was before.

Reply:
I don't feel it's too horrible to have all of these in a single and logical place. It also could have gone into sysctl for the most part, or better, we'd patch the kernel defaults. But all of those risk that it becomes overly complex to sort out power related problems where "something is not tuned right" but there are 20 places it could be.

Having it here also allows us to "just disable clr-power-tweaks" as an initial problem solving step...